tools/cbmc/proofs/run-all-proofs.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
+
+PROOFS_DIR=$(cd "$(dirname "$0")" || exit 1 ; pwd)
+LITANI="${PROOFS_DIR}/../litani/litani"
+
+"${PROOFS_DIR}"/prepare.py
+if [ "$?" -ne 0 ]; then exit 1; fi
+
+"${LITANI}" init --project "Amazon FreeRTOS"
+if [ "$?" -ne 0 ]; then exit 1; fi
+
+N_PROOFS=$(find "${PROOFS_DIR}" -name cbmc-batch.yaml | wc -l)
+CNT=0
+
+echo "Configuring..."
+find "${PROOFS_DIR}" -name cbmc-batch.yaml | while read -r yaml; do
+  PROOF_PATH=$(dirname "${yaml}")
```

Comment:
I've never seen this construction before, what is yam in the loop?

Reply:
This is now reimplemented in python. `yaml` is the variable that gets set in the head of the loop. What this does is set the `yaml` variable to be each line of the `find` command in a loop.

Doing something like this is unsafe:
```sh
for f in $(find . -name *.yaml)
  ...
done
```
because this is looping over every space-separated output of `find`, rather than every line, which is what the `while read` loop does.